worldlimiter.lua

```diff
@@ -1,29 +1,50 @@
+local WorldLimiter_ScheduleInProgress = 0
```

Comment:
![Not impressed](https://cdn.scratch.mit.edu/static/site/projects/thumbnails/195/5210.png)